velox/exec/tests/RowContainerTest.cpp

```diff
@@ -729,6 +729,37 @@ TEST_F(RowContainerTest, rowSizeWithNormalizedKey) {
   ASSERT_EQ(numRows, 1);
 }
 
+TEST_F(RowContainerTest, estimateRowSize) {
+  auto numRows = 1000;
+
+  // Make a RowContainer with a fixed-length key column and a variable-length
+  // dependent column.
+  auto rowContainer = makeRowContainer({BIGINT()}, {VARCHAR()});
+  EXPECT_FALSE(rowContainer->estimateRowSize().has_value());
+
+  // Store rows to the container.
+  auto input = std::dynamic_pointer_cast<RowVector>(
+      BatchMaker::createBatch(ROW({BIGINT(), VARCHAR()}), numRows, *pool_));
```

Comment:
BatchMaker is deprecated. Use VectorMaker instead.

Reply:
Revised